### DISCUSSION TITLE
Get image size (dimensions x and y) and resolution

### DISCUSSION THREAD

**Discussion by bilo2000** [gabrieldwight/Whatsapp-Business-Cloud-Api-Net#85] [General]
Hi,
i was able to recieve and get image file from the client, also i am able to get the file URL, ID and file size in bytes, however, i am still in need to get the following information

- Image dimentisons (High and Width)
- Image resolution 

How can I got that?

**Comment by gabrieldwight** [comment]
Hi, you will have to use your own implementation of platform-specific like system.drawing class to get the image dimensions and resolution or use an open source imaging library to extract the image metadata

**Comment by bilo2000** [comment]
And how to read the image, should i download it first to folder and then read it? can i read it direct from the request?

**Reply by gabrieldwight** [reply]
you can either download the image to a folder or read the image directly from the request with the byte data as long as you handle the image stream and dispose it
